docs/tutorials/jokes.md

```diff
@@ -1582,8 +1582,10 @@ import type { LoaderFunction } from "remix";
 import type { User } from "@prisma/client";
 
 import { db } from "~/utils/db.server";
+import { useLoaderData } from "remix";
```

Comment:
We already have an import statement for `remix` here, so please add `useLoaderData` in that line (L2)

Reply:
It's an `import type`

Reply:
We have

```tsx
// this is just an example. No need to copy/paste this 😄
import { json } from "remix";
import type { LoaderFunction } from "remix";
```

So `useLoaderData` can be added to the `json` import line

Reply:
oh, totally didn't see that

Reply:
@sidv1905 do you mind fixing this last thing? I swear we're gonna get this merged 😅